### ISSUE TITLE
CivilizationCraft Hook?

### ISSUE THREAD

**Issue by xH3LLRAIZ3Rx** [LlmDl/Towny#2396]
is it possible if someone can make me a hook for Towny that hooks in CivilizationCraft, where the only change is War is Toggle-able, Enchanting is like normal and the Economy is though Essentials and the XP drops are Normal compared to Regular CivilizationCraft where XP drops equal Coins for the Economy because I have a Plugin that uses vanilla XP

here is the source for CivilizationCraft: https://github.com/ataranlen/civcraft

**Comment by LlmDl** [comment]
I don't think this will happen. I don't know enough about civcraft to even understand the request sorry.

**Comment by xH3LLRAIZ3Rx** [comment]
basicly I want civcraft and towny to run together in harmony, have the awesome features of civcraft but keep the simplicity of Towny

**Comment by LlmDl** [comment]
That's nice, I still don't understand any of the request. I would need you to explain in great detail exactly you want out of each feature, how the feature works currently, what doesn't work.

**Comment by xH3LLRAIZ3Rx** [comment]
Well what I want is for someone to modify the source of Civcraft (which I supplied in my first comment) to where it works with Towny without any conflicts, and where the Money Making is through Vault and Essentials and not from vanilla EXP Drops (because with Civcraft the EXP that drops from mobs equal money and not actual EXP)
